README.md

```diff
@@ -268,6 +268,10 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // The amount of time before the OIDC Authorization Code grant
                 // times out, requiring another attempt at authentication.
                 "state_timeout_in_seconds": 600
+
+                // The name of the OIDC claim which provides the mapping of a
+                // OIDC user to a irods user account
```

Comment:
"a irods" -> "an iRODS"

Reply:
same with OIDC i think, if we're pronouncing it oh-eye-dee-see.

"a OIDC" -> "an OIDC"

Reply:
oh, reading from bottom to top... i duplicated... carry on...

Reply:
Done.